examples/presentation.ipynb

```diff
@@ -0,0 +1,274 @@
+    "But what if that is not always true?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Imports\n",
+    "\n",
+    "import sys\n",
+    "sys.path.append('..')\n",
+    "from matplotlib.patches import Arc\n",
+    "from matplotlib.pyplot import text, gca\n",
+    "from quantum_compiler.drawing import draw_quantum_state, draw_qubit\n",
+    "from quantum_compiler.qbit import QuantumCircuit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "#### Qubit\n",
+    "A qubit (or quantum bit) is the quantum mechanical analogue of a classical bit.\n",
+    "In classical computing, the information is encoded in bits, where each bit can have the value zero or one.\n",
+    "In quantum computing, the information is encoded in qubits.\n",
+    "A qubit is a two-level quantum system where the two basis qubit states are usually written as:\n",
+    "$ \\ket{0} = \\myvector{1 \\\\ 0} $ and $ \\ket{1} = \\myvector{0\\\\ 1} $.  \n",
+    "\n",
+    "Two other states, called orthogonal x-basis states are\n",
+    "$ \\ket{+} = \\myvector{\\frac{\\sqrt{2}}{2} \\\\ \\frac{\\sqrt{2}}{2}}$ and\n",
+    "$ \\ket{-} = \\myvector{\\frac{\\sqrt{2}}{2} \\\\ \\frac{-\\sqrt{2}}{2}}$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "draw_qubit()\n",
+    "\n",
+    "# Create our first qubit\n",
+    "first_qubit = QuantumCircuit(1)\n",
+    "# Draw it on plane\n",
+    "draw_quantum_state(first_qubit.matrix_representation, name='|0>')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "#### NOT gate\n",
+    "First gate we will talk about is NOT gate, also called Pauli-X gate.\n",
+    "It transforms $ \\ket{0} $ to $ \\ket{1} $ and $ \\ket{1} $ to $ \\ket{0} $.\n",
+    "Looking at it from drawing-on-plane approach, it is a rotation through $\\pi$ radians around x-axis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "draw_qubit()\n",
+    "\n",
+    "# Draw |0> on plane\n",
+    "draw_quantum_state(first_qubit.matrix_representation, name='|0>')\n",
+    "\n",
+    "# Put not gate on qubit on position 0\n",
+    "first_qubit.x(0)\n",
+    "\n",
+    "# Draw |1> on plane\n",
+    "draw_quantum_state(first_qubit.matrix_representation, name='|1>', color='red')\n",
+    "gca().add_patch(Arc((0,0),0.4,0.4,angle=0,theta1=0,theta2=90))\n",
+    "text(0.21,0.09,'x-gate')\n",
+    "\n",
+    "# Put qubit back in |0> state\n",
+    "first_qubit.x(0)"
```

Comment:
it shouldn't be rolled back into |0> state, otherwise, there should be differences below.